variables.tf

```diff
@@ -248,3 +248,12 @@ DOC
 
   default = "aws-cli"
 }
+
+variable "custom_error_response" {
+  # http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/custom-error-pages.html#custom-error-pages-procedure
+  # https://www.terraform.io/docs/providers/aws/r/cloudfront_distribution.html#custom-error-response-arguments
+  description = "(Optional) - List of one or more custom error response element maps"
```

Comment:
Drop `(Optional) - ` prefix as it will be automatically annotated by `terraform-docs`